src/core/Chip/Chip.baseStyles.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme, SuomifiThemeProp } from '../theme';
+import { element, font, focus } from '../theme/reset';
+import { disabledCursor } from '../../components/utils/css';
+
+const removableStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-chip--removable {
+    & .fi-chip--content {
+      max-width: 248px;
+      margin-right: ${theme.spacing.xs};
+    }
+  }
+`;
+
+const disabledStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-chip--disabled.fi-chip {
+    background: ${theme.colors.depthLight1};
+    &:hover,
+    &:active {
+      background: ${theme.colors.depthLight1};
+    }
+    ${disabledCursor}
+  }
+`;
+
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
+    ${focus({ theme })}
+
+    &:focus{
+      &:after {
+      border-radius: 16px;
+      }
+    }
+
+    &.fi-chip {
+      border-radius: 14px;
+      padding: ${theme.spacing.insetXxs} ${theme.spacing.insetL};
+      color: ${theme.colors.whiteBase};
+      background: ${theme.colors.highlightBase};
+      font: ${theme.typography.actionElementInnerText};
```

Comment:
Is the bodyText above for something else or could we just use this one with the font function? And should the font function be used here as well? I think it applies e.g. word-breaks in addition to text design token.

Reply:
Just using actionElementInnerText in the main level font function seems to be enough, since there are no external labels or other things to consider here. Good point.

Reply:
This is now fixed.